Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1307,6 +1314,25 @@ class BrowserViewController: UIViewController {
     }
 }
 
+extension BrowserViewController: ClipboardBarDisplayHandlerDelegate {
+    func shouldDisplayClipboardBar(absoluteString: String) {
+        clipboardToast = ButtonToast(labelText: Strings.GoToCopiedLink, descriptionText: absoluteString,  buttonText: Strings.GoButtonTittle, completion: { (buttonPressed) in
```

Comment:
Done. I'm passing `SettingsDelegate` reference to `ClipboardBarDisplayHandler` and not it can handle everything there and call the delegate that's already implemented :)

Also, I noticed that the `show` method didn't have a way to send the toast duration, so I added the option to send it as a parameter, with a default